tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -270,6 +292,45 @@ def test_ops_integer(get_clients, nr_clients, op_str) -> None:
     assert np.allclose(result, expected_result, atol=10e-3)
 
 
+@pytest.mark.parametrize("nr_clients", [3, 5, 7])
+@pytest.mark.parametrize("op_str", ["add", "sub"])
+def test_ops_integer_falcon(get_clients, nr_clients, op_str) -> None:
```

Comment:
Small question, why is it test_ops_<integer>? A more appropriate name would be something like "test_ops_public_falcon"?